app/Http/Controllers/MailController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Support\Str;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Mail;
+
+class MailController extends Controller
+{
+    public function claim(Request $request)
+    {
+        // Local id and token generation for now.
+        $email = $request->input('email');
+        $lockerid = Str::random(8);
+        $token = Str::random(16);
+        $url = 'https://b-locker.nl/claim/passcode?id=' . e($lockerid) . '&token=' . e($token);
+
+        Mail::send('emails.claim', ['lockerid' => $lockerid, 'url' => $url], function ($message) use($email) {
+
+            $message->to($email);
+            $message->subject('Set passcode');
+        });
+
+        return 'claim sent to '.$email;
+    }
+
+    public function forgot(Request $request)
+    {
+        // Local id and token generation for now.
+        $email = $request->input('email');
+        $lockerid = Str::random(8);
+
+        Mail::send('emails.forgot', ['lockerid' => $lockerid], function ($message) use ($email) {
```

Comment:
I don't want to write too many comments about code style, but use `camelCase` for variables, `snake_case` for array keys.

Reply:
And take care about the new lines.

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA